pkg/helpers/groupsync/groupsyncer.go

```diff
@@ -85,6 +86,10 @@ func (s *LDAPGroupSyncer) Sync() ([]*userv1.Group, []error) {
 		// update the OpenShift Group corresponding to this record
 		openshiftGroup, err := s.makeOpenShiftGroup(ldapGroupUID, usernames)
 		if err != nil {
+			if ldapquery.IsQueryOutOfBoundsError(err) {
+				klog.Warningf("%s", err.Error())
```

Comment:
We rarely use `klog` for these warnings, `fmt.Fprintf(os.Stderr` is the preferred.

Reply:
updated